tests/bootstrap.php

```diff
@@ -1,6 +1,7 @@
 <?php
-require_once __DIR__.'/../vendor/autoload.php';
 
-if (!class_exists('\PHPUnit\Framework\TestCase') && class_exists('\PHPUnit_Framework_TestCase')) {
-    class_alias('\PHPUnit_Framework_TestCase', '\PHPUnit\Framework\TestCase');
-}
+    require_once __DIR__.'/../vendor/autoload.php';
+
+//    if (!class_exists('\PHPUnit\Framework\TestCase') && class_exists('\PHPUnit_Framework_TestCase')) {
```

Comment:
Why is this commented out? It's here to support older PHPUnit version? if it's not needed, we should drop it completely (instead of just commenting out).